examples/dominant-assurance/index.rsh

```diff
@@ -105,8 +121,11 @@ export const main = Reach.App(() => {
                                  - investorFailProfit * numInvestors;
     transfer(returnedToEntrepreneur).to(E);
```

Comment:
You should have some events in this program, probably one here so you can have the investors synchronize on it